README.md

```diff
@@ -87,6 +88,45 @@ Ejecting `@dojo/cli-build-webpack` will produce a `config/build-webpack/webpack.
 node_modules/.bin/webpack --config=config/build-webpack/webpack.config.js
 ```
 
+### Interop with external libraries
+
+External libraries that can not be loaded normally via webpack can be included in a Dojo 2 providing an implementation of `require` or `define`, and some
+configuration in the project's `.dojorc` file.
+`.dojorc` is a JSON file that contains configuration for Dojo 2 CLI tasks. Configuration for the `dojo build` task can be provided under the
+`build-webpack` property.
+Configuration for external dependencies can be provided under the `externals` property of the `build-webpack` config.
+
+`externals` is an object, with two allowed properties.
+
+ `outputPath`: An optional property specifying an output path to which files should be copied.
+
+ `dependencies`: A required array that defines which modules should be loaded via the external loader, and what files should be included in the build. Each entry can be one of two types:
+* A string. In this case the entry simply indicates that this path, and any children of this path, should be loaded via the external loader
+* An object that provides additional configuration for dependencies that need to be copied into the built application. This object has the following properties:
+
+ | Property | Type | optional | Description |
+ | -------- | ---- | -------- | ----------- |
+ | `from` | `string` | false  | A path relative to `node_modules` that tells the loader where it can find this dependency in order to copy it into the built application |
```

Comment:
"A path relative to `node_modules` that tells the loader where it can find this dependency in order to copy it into the built application" -> "A path relative to `node_modules` specifying the dependency location to copy into the built application" ?